components/heading/tests/unit/heading.spec.js

```diff
@@ -1,3 +1,43 @@
-test('TODO replace this', () => {
-	expect(true).toBe(true);
+import React from 'react';
+import { Heading } from '@westpac/heading';
+import { render } from '@testing-library/react';
+
+import { overridesTest } from '../../../../helpers/tests/overrides-test.js';
+import { nestingTest } from '../../../../helpers/tests/nesting-test.js';
+import { GEL } from '@westpac/core';
+import wbc from '@westpac/wbc';
+// import { Heading } from '@westpac/button-dropdown';
+
+// The default tests every component should run
+overridesTest({
+	name: 'heading', // the name has to be the package name without '@westpac/' scope
+	overrides: ['Heading'], // every single override root key
```

Comment:
It seems that we are missing an override here: BrandHeading. Check the components/heading/src/overrides folder. You can also compare this with the Alert tests.

Reply:
overrides: ['BrandHeading', 'Heading'], // every single override root key
This is making the unit test fail. I need to discuss this with you further.

Reply:
Ok, now the branch includes testing for BrandHeading component.